src/app/mainview/mainview.component.ts

```diff
@@ -1389,6 +1391,19 @@ export class MainviewComponent implements OnInit {
         return count;
     }
 
+    public getUnusedRegRegions(){
+        this.regressionRegions.forEach(x=> {
+            this.scenarios.forEach(s=> {
+                s.regressionRegions.forEach(rr => {
+                    if (x.id == rr.id) {
+                        this.regRegionsScenarios.push(x)
+                    }
+                });
+            });
+        });
+        this.unusedRegRegions = this.regressionRegions.filter(entry1 => !this.regRegionsScenarios.some(entry2 => entry1.id === entry2.id));
```

Comment:
This could probably slimmed down a bit using nested filters, but it works so I'm ok with it 👍